### PR TITLE
Basic Log In

### PR DESCRIPTION
**Pull request by syahidazuri**

### CODE REVIEW

**Comment by syahidazuri** [comment]
Ready

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (manhnt-0960) [inline app/assets/stylesheets/sessions.scss]: delete this empty file

File: app/assets/javascripts/sessions.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (manhnt-0960) [inline app/assets/javascripts/sessions.js]: similar

File: app/models/user.rb [hunk 3]
```diff
@@ -1,11 +1,29 @@
...
+    def digest string
+      cost = ActiveModel::SecurePassword.min_cost ? Bcrypt::Engine::MIN_COST :
+                                                  Bcrypt::Engine.cost
+      Bcrypt::Password.create string, cost: cost
+    end
+  end
+
+  private
+
+
```
Comment (manhnt-0960) [inline app/models/user.rb]: remove this empty line

File: app/models/user.rb [hunk 4]
```diff
@@ -1,11 +1,29 @@
...
+    length: { maximum: Settings.validations.email.max_length },
+    format: { with: Settings.validations.email.regex },
+    uniqueness: true
+     
+  validates :password, presence: true, 
+    length: { minimum: Settings.validations.password.min_length }, allow_nil: true
 
   has_secure_password
+
+  class << User
```
Comment (manhnt-0960) [inline app/models/user.rb]:
```suggestion
  class << self
```

**Comment by syahidazuri** [comment]
Ready

**Pull request merged**